### DISCUSSION TITLE
Whisper model is unable to segment sentences after LoRA finetune

### DISCUSSION THREAD

**Discussion by bardenthenry** [openai/whisper#1714] [General]
I have about 90,000 records of data from Common Voice, and approximately 10,000 records from our own recorded meetings. I performed finetuning on Whisper Large-V2 using LoRA, and accelerated it using the ct2-transformers-converter. However, the finetuned Whisper model is unable to segment sentences. How can I adjust to resolve this problem?

**Comment by casic** [comment]
Hi,  what hardware is needed for  fine-tune   large-v2 ??   I have tried
with   3  Tesla T4 16GB   , but only  small model running.    Medium and
large-v2  was out of memory :(.  Please  give minimal configuration to fine
tune  whisper  large-v2.  Thanks

На пн, 16.10.2023 г. в 5:15 ч. bardenthenry ***@***.***>
написа:

**Reply by bardenthenry** [reply]
Hi @casic.
My hardware is just one RTX 4090, AMD 7950X and 32G DRAM.
